pyiode/objects/table.pyx

```diff
@@ -22,8 +22,14 @@ cdef class TableCell:
 
     Attributes
     ----------
+    # GDM>
+    # * I *wonder* if using StringCell and LecCell subclasses wouldn't be better (for example if LecCell had additional methods)?
```

Comment:
Users are not supposed to create TableCell instances themselve. TableCell instances are created from the class TableLine. Idem for TableLine instances, they are only created from methods of the Table class.

I am trying to stay as close as possible the C and C++ API, so I will keep the TableCell for the moment. I could change my mind in the future when I will have more time.

But thanks for the suggestion.

Reply:
See issue #582